src/main/java/org/folio/circulation/domain/RequestServiceUtility.java

```diff
@@ -133,15 +160,17 @@ static Result<RequestAndRelatedRecords> refuseWhenUserIsInactive(
   static Result<RequestAndRelatedRecords> refuseWhenMovedToDifferentInstance(
     RequestAndRelatedRecords requestAndRelatedRecords) {
 
+    log.debug("refuseWhenMovedToDifferentInstance:: parameters requestAndRelatedRecords: {}",
+      () -> requestAndRelatedRecords);
     Request request = requestAndRelatedRecords.getRequest();
     Item item = request.getItem();
     if (!Objects.equals(item.getInstanceId(), request.getInstanceId())) {
       HashMap<String, String> parameters = new HashMap<>();
       parameters.put(ITEM_ID, request.getItemId());
       parameters.put(INSTANCE_ID, item.getInstanceId());
-      return failedValidation(
-        new ValidationError("Request can only be moved to an item with the same instance ID",
-          parameters));
+      String message = "Request can only be moved to an item with the same instance ID";
+      log.warn("refuseWhenMovedToDifferentInstance:: {}", message);
```

Comment:
This UI-oriented error message does not make much sense in the logs. Can we say something more related to the check we are performing? Like "requested instance ID does not match item's instance ID"